openwisp_users/tests/test_admin.py

```diff
@@ -169,6 +169,21 @@ def test_admin_change_user_email_empty(self):
         self.assertEqual(len(mail.outbox), 0)
         self.assertContains(response, 'errors field-email')
 
+    def test_admin_change_user_page_get_invalid_UUID(self):
+        admin = self._create_admin()
+        self.client.force_login(admin)
+        response = self.client.get(
+            reverse(f'admin:{self.app_label}_user_change', args=['WRONG'])
+        )
```

Comment:
It is generally a good practice to keep testing code and assertions together. 

In this case, we can use [`subTest`](https://www.caktusgroup.com/blog/2017/05/29/subtests-are-best/) to further improve the readability. I would structure this test like below:

```python
with self.subTest('Test for wrong identifier'):
    response = self.client.get(
            reverse(f'admin:{self.app_label}_user_change', args=['WRONG'])
    )
    self.assertEquals(response.status_code, 404)
``` 

Similarly, you can add a subTest for the other case, `Test for non-existing user`.